app/models/tortoise/user.py

```diff
@@ -0,0 +1,99 @@
+"""
+This module is made to provide different user models to the API.
+It contains pydantic and Tortoise models.
+"""
+from typing import Annotated, Optional
+
+import bcrypt
+from fastapi import Depends, HTTPException
+from fastapi.security import OAuth2PasswordBearer
+from passlib.context import CryptContext
+from tortoise import fields
+from tortoise.models import Model
+
+from app.models.pydantic.TokenModel import PydanticToken
+from app.utils.http_errors import ClassicExceptions
+
+pwd_context   = CryptContext(schemes=["bcrypt"], deprecated="auto")
+oauth2_scheme = OAuth2PasswordBearer(tokenUrl="/auth/login")
+
+
+class UserInDB(Model):
+    """
+    This class is designed to design the table inside of the database.
+    It contains all the columns necessary to store the user.
+    """
+    id = fields.IntField(primary_key=True)
+    login = fields.CharField(unique=True, required=True, max_length=128)
+    firstname = fields.CharField(max_length=128)
+    lastname = fields.CharField(max_length=128)
+    mail = fields.CharField(unique=True, required=True, max_length=128)
+    hash = fields.TextField(required=True)
+
+    def __str__(self):
+        """
+        This method will output a str describing the UserInDB class.
+        """
+        return "[INFO] - USER " + self.login + " ID : " + str(self.id)
```

Comment:
should use f-string instead like `` f"[INFO] - USER {self.login} ID : {self.id}"``